src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
According to the BEM methodology, modifiers should be used with the block or element name, not alone. Ensure that your class naming follows the BEM convention properly.

Reply:
this are name conventions that are used in tests. Such names are expected in the task